README.md

```diff
@@ -15,54 +15,69 @@
 
 API application template for Yii 3.
 
+## Requirements
+
+- PHP 8.1 or higher.
+
 ## Installation
 
 Creating a project:
 
-```
+```shell
 composer create-project yiisoft/app-api --stability=dev myproject
 cd myproject
 ```
 
 Install docker:
 
-```
+```dockerfile
 docker-compose up -d
 ```
 
 Enter into the container:
 
-```
+```dockerfile
 docker-compose exec php bash
 ```
 
 Install packages:
 
-```
+```shell
 composer install
 ```
 
-Usually the application is available at http://localhost:8080.
+Usually the application is available at <http://localhost:8080>.
 
 Authorization is performed via the `X-Api-Key` header.
 
-## API documentation
+## Documentation
 
-API documentation is available at `/docs`. It is built from OpenAPI annotations (`@OA`).
```

Comment:
Why removing it?

Reply:
Good question.... I think it's because the '/docs' folder only has one file, 'internals.md'. So I thought it was strange to direct it there.
But I will undo the change.